src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/Resource.cs

```diff
@@ -0,0 +1,80 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
```

Comment:
The resource type should not be geenrated into your code.  This is a swagger modeling error